src/cloudflare/internal/d1-api.ts

```diff
@@ -276,7 +276,7 @@ class D1PreparedStatement {
     )
   }
 
-  public async raw<T = unknown>(): Promise<T[]> {
+  public async raw<T = unknown[]>(): Promise<T[]> {
```

Comment:
So, to be clear, the return value is a promise that resolves to an array of arrays, which by default will be `unknown[][]`. Right?

Reply:
That's right, yep.